com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/SyncFlowableStep.java

```diff
@@ -113,7 +118,18 @@ protected void onStepError(DelegateExecution context, Exception e) throws Except
 
     protected void processException(Exception e, String detailedMessage) throws Exception {
         e = handleControllerException(e);
-        throw getExceptionConstructor(e).apply(e, detailedMessage + ": " + e.getMessage());
+        String tail = computeTail(e);
+        throw getExceptionConstructor(e).apply(e, detailedMessage + ": " + e.getMessage() + " " + tail);
```

Comment:
Invoke the method directly in apply() without a new variable.
Rename the method to be related to "support incident hint"